### PR TITLE
implement list mvt mapping

### PR DESCRIPTION
**Pull request by pawelzwolenskicognite**
## Checklist

- [x] Changelog entry added in `CHANGELOG` or not relevant for this change
- [x] Version updated in `pyproject.toml` or not relevant for this change

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [0.71.2]
+
+### Added
+
+- Geospatial: list MVT mappings definitions
```
Comment (tuanng-cognite) [inline CHANGELOG.md:19]: 0.72.0 -> New feature.

File: cognite/experimental/_api/geospatial.py [hunk 2]
```diff
@@ -297,3 +297,22 @@ def retrieve_mvt_mappings_definitions(self, external_id: Union[str, List[str]] =
...
             wrap_ids=True, external_ids=external_id, resource_path=resource_path, cls=MvpMappingsDefinitionList
         )
+
+    @_with_cognite_domain
+    def list_mvt_mappings_definitions(self) -> MvpMappingsDefinitionList:
+        """`List MVP mappings definitions`
+        <https://pr-1653.specs.preview.cogniteapp.com/v1.json.html#operation/GeospatialListMvtMappings>
+
+        Returns:
+            MvpMappingsDefinitionList: the requested mappings or None if it does not exist.
```
Comment (tuanng-cognite) [inline cognite/experimental/_api/geospatial.py:307]: `None` -> should be EmptyList, right?

File: CHANGELOG.md (hunk 1 above)
Comment (pawelzwolenskicognite) [reply CHANGELOG.md:19]: sorry, but I don't understand that comment :)

Comment (tuanng-cognite) [reply CHANGELOG.md:19]: semantic version is used in this repo. New feature means that we bump the minor version.
In our case, 0.71.1 -> 0.72.0

Comment (pawelzwolenskicognite) [reply CHANGELOG.md:19]: ah ok, fixed :)

**Review by tuanng-cognite — APPROVED** [review]

**Pull request merged**